build.gradle

```diff
@@ -100,6 +100,9 @@ allprojects {
         maven {
             url "https://maven.research.rackspacecloud.com/content/groups/aggregate/"
         }
+        maven {
+            url 'https://build.shibboleth.net/nexus/content/repositories/releases'
```

Comment:
I immediately start to question things when they aren't in any of the big repos. Not being in central is a little sketch not being in some of the others like jcenter is super sketch.